controlplane/src/core/build-server.ts

```diff
@@ -182,6 +188,13 @@ export default async function build(opts: BuildConfig) {
     mailerClient = new Mailer({ username: opts.smtpUsername, password: opts.smtpPassword });
   }
 
+  // required to verify webhook payloads
+  await fastify.register(import('fastify-raw-body'), {
```

Comment:
Weird, that this is not supported nativly.

Reply:
yeah took me a bit to figure out, but not possible without a plugin.